src/TrackerCouncil.Smz3.Data/Options/GeneralOptions.cs

```diff
@@ -181,6 +181,11 @@ public string? TwitchId
     /// </summary>
     public bool DisplayTrackerSpeechWindow { get; set; }
 
+    /// <summary>
+    /// If the gRPC server for receiving MSU randomizer messages should be disabled
+    /// </summary>
+    public bool DisableMsuMessageReceiver { get; set; }
```

Comment:
I'm usually in favor of naming things positively (`EnableMsuMessagerReceiver`) and then defaulting them to `true` instead, just for clarity's sake.

I'm not sure whether this should be opt-in or opt-out by default, but that's honestly besides my point.

Reply:
Sure, I can make that change for the setting name.

In terms of it this should be enabled by default, I did put some thought into it because of it spinning up a grpc server that can accept messages. 

Obviously there could be security or even privacy concerns just by the mere mention of this spinning up something like this. Ultimately I came to the thought that it would be okay enabled by default given the pretty narrow scope of it. It's only used to give random specifically defined lines in the config or just update a flag in memory.

If concerned, we could possibly give a prompt asking that first time generating a seed without an MSU associated, since that's the only time this is used?

Reply:
Oh, I was more thinking along the lines of this only being applicable when using the standalone MSU randomizer.